ARGO/src/Game.cpp

```diff
@@ -32,6 +35,8 @@ Game::Game()
 		// game doesnt run
 		m_isRunning = false;
 	}
+
+	setupLevel();
```

Comment:
Dont put this here, put it inside the try statement

Reply:
Done